### PR TITLE
mention ivory in the readme

### PR DESCRIPTION
**Pull request by dbu**
to avoid confusion. ideally, @egeloen would add some note in ivory about this as well. (for now maybe with a warning that this one is not yet ready for production)

### CODE REVIEW

**Comment by sagikazarmark** [comment]
Be careful with the name 'Ivory'. It is a vendor name as well, not just the name for the HTTP Adapter package.

**Comment by dbu** [comment]
better?

**Comment by willdurand** [comment]
:+1:

**Comment by egeloen** [comment]
I just added a note in the readme about the deprecation and these set of packages.

**Comment by egeloen** [comment]
I think I will deprecate it officially in composer when the 1.0 is out here.

File: README.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ $ composer require php-http/httplug
 
 This is the contract package for HTTP Client. Use it to create HTTP Clients which are interoperable and compatible with [PSR-7](http://www.php-fig.org/psr/psr-7/).
 
+This library is the official successor of the [ivory http adapter](https://github.com/egeloen/ivory-http-adapter) and the code is based on the ivory http adapter code.
```
Comment (sagikazarmark) [inline README.md]: Ivory http adapter and code expressions are repeated here. Can we make it simpler? Also, the code is completely refactored. Actually it has been refactored several times. I think it is enough to mention that the idea is from ivory, it evolved from an adapter package to a PSR-7 HTTP Client abstraction and it is a successor of IHA.

**Comment by dbu** [comment]
thanks a lot egeloen!

i updated the text a bit. ok @sagikazarmark ?

**Comment by sagikazarmark** [comment]
:+1:

**Pull request merged**